pages/proposal/[id].tsx

```diff
@@ -37,6 +37,36 @@ const ProposalItemDetail = () => {
 				},
 			})
 
+			const proposalVotes = await near.nearViewFunction({
+				contractName: CONTRACT.DAO,
+				methodName: 'get_proposal_votes',
+				args: {
+					id: parseInt(router.query.id as string),
+					from_index: 0,
+					limit: 10,
+				}
+			})
+
+			const proposalVoteUser = await near.nearViewFunction({
+				contractName: CONTRACT.DAO,
+				methodName: 'get_proposal_vote',
+				args: {
+					id: parseInt(router.query.id as string),
+					account_id: accountId
+				}
+			})
+
+			const proposalVotesWrap: IVotes = {}
+
+			proposalVotes.forEach((userVoteTuple: Array<any>) => {
+				proposalVotesWrap[userVoteTuple[0]] = userVoteTuple[1]
+			})
```

Comment:
bisa gini sih, biar enak instead of `Array<any>`
```tsx
proposalVotes.forEach(([accountId, accountVotes]: [string, IUserVote]) => {
   proposalVotesWrap[accountId] = accountVotes
})

```

Reply:
atau opsi lain pas viewFunctionnya kita define typenya
```tsx
const proposalVotes: [string, IUserVote][] = await near.nearViewFunction({
```